### ISSUE TITLE
Broken link in core/howto/tutorial/intro

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4201]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4201|
|Type|enhancement|
|Created|2010-01-05 23:16:23Z|

link target is "glossary.xhtml#Daemon"

should be: "../glossary.xhtml#Daemon"

patch forthcoming

Attachments:

* [4201_tutorial_bad_link_fix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/48c/48c2baaa09626ddda596a4f1307ea7d63b5f2f43/f518f30108b235d74f57291b6878011a1c103e51.patch) (704 bytes) - added by khorn on 2010-01-05 23:17:39Z - Patch to fix bad link in core/howto/tutorial/intro

<details><summary>Searchable metadata</summary>

```
trac-id__4201 4201
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1262733383000000 1262733383000000
changetime__1262750028000000 1262750028000000
version__None None
owner__ 
cc__khorn cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_1"></a>|@khorn removed owner|
|-|-|

Patch attached, sending to review...

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess commented|
|-|-|

Good catch - applying now.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess set status to `closed`|
|-|-|

(In [27914]) Apply 4201_tutorial_bad_link_fix.patch

Author: khorn Reviewer: jesstess Fixes: [#4201](https://github.com/twisted/twisted/issues/4201)

Fix broken link to glossary term.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
